docker/maintenance/cronjob-generator.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+## usage ./cronjob-generator
+SCRIPT_FOLDER=$(dirname $(readlink -f $0))
+JOB_DESCRIPTIONS=(`ls $SCRIPT_FOLDER | grep -v 'example' | grep -v 'cronjob-template.yaml' | grep '.yaml'`)
+TEMPLATE=`cat $SCRIPT_FOLDER/cronjob-template.yaml`
+
+for job_description_file_name in "${JOB_DESCRIPTIONS[@]}"; do
+	JOB_JSON=`js-yaml $SCRIPT_FOLDER/$job_description_file_name`
+	NAME=`echo "$JOB_JSON" | jq .name`
```

Comment:
you can use jq to extract all of this at once
fe `js-yaml founder-emails-complete-digest.yaml | jq '(.name | tostring)  + (.schedule | tostring)'`

Reply:
neat, did not know that
but then it gives me a long string that I have to cast to an array I guess?
and then use array in script?